src/brs/Burst.java

```diff
@@ -41,10 +42,13 @@
 
 public final class Burst {
 
-  public static final Version VERSION = Version.parse("v2.4.2");
+  public static final Version VERSION = Version.parse("v2.5.0");
```

Comment:
Logic to handle the disconnection from old nodes after the fork. So we have VERSION and MIN_VERSION as variables.